protocols/util/avro_factory.py

```diff
@@ -0,0 +1,57 @@
+import sys
+import factory.fuzzy
+from factory.base import FactoryMetaClass, BaseFactory, BaseMeta
+
+
+def mock_schema(protocol):
+    """
+    :type protocol: Protocol
+    """
+    INT_MIN_VALUE = -(1 << 31)
+    INT_MAX_VALUE = (1 << 31) - 1
+    LONG_MIN_VALUE = -(1 << 63)
+    LONG_MAX_VALUE = (1 << 63) - 1
+    expected_schema = protocol.schema
+
+    mock_result = {}
+    for field in expected_schema.fields:
+        schema_type = field.type.type
+        name = field.name
+        if schema_type == 'null':
+            mock_result[name] = 'null'
+        elif schema_type == 'boolean':
+            mock_result[name] = factory.fuzzy.FuzzyChoice([True, False])
+        elif schema_type == 'string':
+            mock_result[name] = factory.fuzzy.FuzzyText()
+        elif schema_type == 'bytes':
+            mock_result[name] = b'a'
+        elif schema_type == 'int':
+            mock_result[name] = factory.fuzzy.FuzzyInteger(INT_MIN_VALUE, INT_MAX_VALUE)
+        elif schema_type == 'long':
+            mock_result[name] = factory.fuzzy.FuzzyInteger(LONG_MIN_VALUE, LONG_MAX_VALUE)
+        elif schema_type in ['float', 'double']:
+            mock_result[name] = factory.fuzzy.FuzzyFloat(sys.float_info.min)
+        elif schema_type == 'enum':
+            mock_result[name] = factory.fuzzy.FuzzyChoice(expected_schema.symbols)
```

Comment:
When you add support for arrays, the podam library we used in Java creates by default all lists of length 5. If you want to customise length of a given array you can only specify in an ad hoc parameter in kwargs.